### ISSUE TITLE
Unresolved declaration file for vue3-simple-icons.mjs

### ISSUE THREAD

**Issue by Tbaile** [wyatt-herkamp/vue3-simple-icons#6]
Hello!

It seems that I've found an issue when importing the library in a new Vue + Vite project. Seems that in the `package.json` there's no export of types for the `node_modules/vue3-simple-icons/dist/vue3-simple-icons.mjs` file.

The only thing missing is the `types` entry under the `exports` in the `package.json` as below:
```
"exports": {
    ".": {
      "import": "./dist/vue3-simple-icons.mjs",
      "require": "./dist/vue3-simple-icons.umd.js",
      "types": "./dist/types/main.d.ts"
    }
  }
  ```

This solves the issue and typescript can import types correctly, feel free to ask for a PR on this.

Thank you! :beers:

**Comment by wyatt-herkamp** [comment]
Thx!. I will push this fix later. It will not be released till Sunday.

**Comment by Tbaile** [comment]
No problem, thanks!

**Comment by wyatt-herkamp** [comment]
@Tbaile this should be fixed in 11.0.0
